SPECS/dwz/dwz.spec

```diff
@@ -1,12 +1,12 @@
 Summary:        DWARF optimization and duplicate removal tool
 Name:           dwz
-Version:        0.13
-Release:        4%{?dist}
+Version:        0.14
+Release:        1%{?dist}
 License:        GPLv2+ and GPLv3+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://sourceware.org/dwz
-Source:         https://sourceware.org/ftp/dwz/releases/%{name}-%{version}.tar.xz
+Source:         https://sourceware.org/ftp/dwz/releases/%{name}-%{version}.tar.xz#/%{name}-%{version}.tar.xz
```

Comment:
Nit: We don't need the `#/%{name}-%{version}.tar.xz` part here, since the downloaded file is already in that form